docs(cli): add docs for custom plugins (#312)
code-pushup/cli: examples/plugins/src/file-size/README.md

# file-size-plugin

> 🕵️ **Code PushUp plugin for detecting changes in filesize using different compressions** 📋
> 
> ---
> 
> The plugin crawls the file base depending on your configuration and reports about their filesize.

Reviewer: ```suggestion
🕵️ **Code PushUp plugin for detecting changes in file size using different compressions** 📋

---

The plugin crawls the file base depending on your configuration and reports about their file size.
```

# file-size-plugin

🕵️ **Code PushUp plugin for detecting changes in file size using different compressions** 📋

---

The plugin crawls the file base depending on your configuration and reports about their file size.

You can configure the plugin with the following options:

- directory to crawl
- file name pattern
- budget as number in bytes

## Getting started

1. If you haven't already, install [@code-pushup/cli](../cli/README.md) and create a configuration file.

2. Copy the [plugin source](../file-size) as is into your project

3. Add this plugin to the `plugins` array in your Code PushUp CLI config file (e.g. `code-pushup.config.js`).

>    Pass in the path od the directory to crawl (relative to `process.cwd()`), as well as patterns and a budgget.

Reviewer 2: ```suggestion
   Pass in the path on the directory to crawl (relative to `process.cwd()`), as well as patterns and a budget.
```

   ```js
   import fileSizePlugin from './file-size.plugin';

   export default {
     // ...
     plugins: [
       // ...
       fileSizePlugin({
         directory: 'dist',
         patterns: /.js$/,
         budget: 42000,
       }),
     ],
   };
   ```

> 4. (Optional) Reference audits (or groups) which you wish to include in custom categories (use `npx code-pushup print-config` to list audits and groups).

Reviewer 2: ```suggestion
4. (Optional) Reference audits (or groups) that you wish to include in custom categories (use `npx code-pushup print-config` to list audits and groups).
```
